Description: Drop appmenu workaround.
 It causes 100% CPU usage when using appmenu.
Author: Benjamin Drung <bdrung@ubuntu.com>
Bug-Ubuntu: https://launchpad.net/bugs/996407
Forwarded: no

--- a/wx/src/gtk/frame.cpp
+++ b/wx/src/gtk/frame.cpp
@@ -562,16 +562,6 @@
     if (m_frameMenuBar)
     {
         m_frameMenuBar->OnInternalIdle();
-
-        // UpdateMenuBarSize may return a height of zero on some
-        // systems (e.g. Ubuntu 11.04 Alpha as of 2010-12-06),
-        // when the menubar widget has not been fully realized.
-        // Update the menu bar size again at this point, otherwise
-        // the menu would not be visible at all.
-        if (!m_menuBarHeight)
-        {
-            UpdateMenuBarSize();
-        }
     }
 #endif // wxUSE_MENUS_NATIVE
 #if wxUSE_TOOLBAR
